ts/receiver/contentMessage.ts

```diff
@@ -412,6 +423,7 @@ export async function innerHandleContentMessage(
 
     if (content.preKeyBundleMessage) {
       await handleSessionRequestMessage(envelope, content.preKeyBundleMessage);
+      return;
```

Comment:
Why are we returning here? I'm pretty sure we have a message somewhere which is a session request with some content in it